.github/workflows/workflow.yml

```diff
@@ -131,7 +131,9 @@ jobs:
       run: |
         TAG="$(CI_FORCE_CLEAN=1 bin/root-tag)"
         export KIND_CLUSTER=github-$TAG-${{ matrix.integration_test }}
-        bin/kind create cluster --name=$KIND_CLUSTER --wait=1m
+        # retry cluster creation once in case of port conflict or kubeadm failure
+        bin/kind create cluster --name=$KIND_CLUSTER --wait=2m --loglevel debug ||
+          bin/kind create cluster --name=$KIND_CLUSTER --wait=2m --loglevel debug
```

Comment:
If the first cluster creation fails, will the cluster remain on the host? Or does `kind` perform auto cleanup? If it doesn't clean up after itself, the second attempt to create the cluster will fail, right? In that case, I wonder if we can just do a `docker restart $KIND_CLUSTER-control-plane`?

Reply:
Yeah, `kind` cleans up after itself if cluster creation fails.